hack/install-cloud-tools.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
I'll repeat my comment from the other PR. I don't see why this is useful. Running `./hack/eks` (although should probably be renamed to `./hack/terraform-eks[.sh]` is perfectly fine IMO and I'm not sure what moving things around buys you.

Reply:
it buys not a lot. I do it because I am working on multiple branches, this allows me to install tools to setup EKS independent of EKS.

Reply:
on second thought, I will modify cloud.md refers to eks script as ./hack/terrafrom-eks.sh, this makes cloud.md Antrea centric.

On the other hand, I prefers to leave install-cloud-tools.sh as it is for convenience. Hope this works for you.

Reply:
We can keep the script, as long as there is no Makefile target any more